go.mod

```diff
@@ -34,4 +34,6 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.3
 )
 
+replace github.com/codeready-toolchain/toolchain-common => github.com/xcoulon/toolchain-common v0.0.0-20210914134654-45c375935ced
```

Comment:
What do you need this for?

Reply:
there are similar changes in https://github.com/codeready-toolchain/toolchain-common/pull/210 and I think we should build/test on those, too.

Reply:
I updated the description to mention this dependency